board/piksiv3/rootfs-overlay/etc/init.d/S84piksi_fpga

```diff
@@ -8,7 +8,11 @@ start() {
     cat "$bit" > /dev/xdevcfg
     prog_done=`cat /sys/devices/soc0/amba/f8007000.devcfg/prog_done`
     if [ "$prog_done" != "1" ]; then
-      echo "ERROR: configuration failed"
+      while [ 1 ]; do
```

Comment:
The firmware already loops waiting for the FPGA if it is not configured. I recommend not blocking here.